src/rendition-mixin.js

```diff
@@ -66,6 +66,12 @@ class Representation {
 
     this.bandwidth = playlist.attributes.BANDWIDTH;
 
+    if (playlist.attributes.CODECS) {
```

Comment:
For DASH content, do we want to check for the CODECS of the audio playlist (if available)?